frontend/src/utils/multiLanguage.ts

```diff
@@ -16,3 +20,64 @@ export const convertUnicodeLocaleToLanguage = (language: Language) => {
 }
 
 export const getDefaultSupportedLanguages = () => Object.values(Language)
+
+interface SelectedLanguageProps {
+  defaultValue: string
+  translations?: TranslationMapping[]
+  selectedLanguage: Language
+}
+
+interface SelectedLanguageFieldOptionProps {
+  defaultValue: string[]
+  translations?: TranslationOptionMapping[]
+  selectedLanguage: Language
+}
+
+export const getValueInSelectedLanguage = ({
+  defaultValue,
+  translations,
+  selectedLanguage,
+}: SelectedLanguageProps) => {
+  let title = defaultValue
```

Comment:
/supernit Why `title`? This function is trying to get the `value` but this suggest that the value is used for `titles`